Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/WebAppPerformanceCollector.cs

```diff
@@ -86,28 +86,16 @@ public void RefreshCounters()
             bool isCustomCounter,
             out string error,
             bool blockCounterWithInstancePlaceHolder)
-        {
-            error = null;
-
+        {            
             try
             {
-                bool useInstancePlaceHolder = false;
-                string parsingError = null;
-                var pc = PerformanceCounterUtility.CreateAndValidateCounter(perfCounter, null, null, out useInstancePlaceHolder, out parsingError);
-
-                if (!string.IsNullOrEmpty(parsingError))
-                {
-                    error = parsingError;
-                }
-
+                bool useInstancePlaceHolder = false;                
+                var pc = PerformanceCounterUtility.CreateAndValidateCounter(perfCounter, null, null, out useInstancePlaceHolder, out error);
+                
                 if (pc != null)
                 {
                     this.RegisterPerformanceCounter(perfCounter, this.GetCounterReportAsName(perfCounter, reportAs), pc.CategoryName, pc.CounterName, pc.InstanceName, useInstancePlaceHolder, false);
-                }
-                else
-                {
-                    this.RegisterPerformanceCounter(perfCounter, this.GetCounterReportAsName(perfCounter, reportAs), string.Empty, perfCounter, string.Empty, useInstancePlaceHolder, false);
```

Comment:
got it. I will bring back the original behavior and add a comment explaining this.